prac_07/README.md

```diff
@@ -0,0 +1,29 @@
+# Prac 7
+## Programming Languages
+The programing languages part of Practical 7.
+Prints out the programming languages in a file called languages.csv
+
+## More Guitars
+The More Guitars Task of Practical 7.
+Displays current guitars in a guitars.csv file.
+Asks user to add new guitars until a blank name is entered, updating the file.
+
+## Project Management Program
+Project Management program from practical 7.
+Displays a menu that the user is able to access.
+L-load: loads a file, default file is projects.txt
+S-save: saves current projects to a specified file.
+D-display: displays the current projects.
+F-filter: shows projects started after a specified date.
+A-add: Adds a project to the current list of projects.
+U-update: Updates an existing project.
+Q-quit: Stops the program from running.
+
+## Lessons Learned:
+- Naming of functions
+- Naming of Variables
+- Specific menu formatting
+
+## Links
+[Programming Patterns](https://github.com/CP1404/Starter/wiki/Programming-Patterns)
+[CP1404 Practicals Instructions Repo](https://github.com/CP1404/Practicals/tree/master/prac_07)
```

Comment:
this is a good idea using hyperlinks 👍 , honestly didn't even think of this 
i dont know if it is just me but the links appear on the same line, if this isnt just me maybe separating the links with a comma or something